### ISSUE TITLE
Syntax error GraphQL (Unexpected EOF) when adding LaravelServiceProvider to config/app.php

### ISSUE THREAD

**Issue by soosap** [nuwave/laravel-graphql-relay#16]
The moment I add `Nuwave\Relay\LaravelServiceProvider::class,` to `app/config.php` I am no longer able to view my application routes via `php artisan route:list`.

I receive the following error:

```
[GraphQL\SyntaxError]
Syntax Error GraphQL (1:1) Unexpected EOF
1:
   ^
```

Steps to reproduce error:
- Set up fresh Laravel install
- Add `Nuwave\Relay\LaravelServiceProvider::class,` to `app/config.php`
- Run `php artisan route:list` to list your application's routes

So I did some error tracing and found that there is a problem in ... `LaravelController`'s **setupQuery**. In particular, the error is caused when `$request->get('query')` results in NULL. I did this simple `is_null` catch to get around this, but I am not really sure what is going on there.

``` php
    public function setupQuery(Request $request)
    {
        $relay = app('relay');

        if (!is_null($request->get('query'))) {
            $relay->setupRequest($request->get('query'));

            foreach ($relay->middleware() as $middleware) {
                $this->middleware($middleware);
            }
        }

    }
```

**Comment by davidstoker** [comment]
+1. I'm experiencing this issue as well and came to the same conclusions

**Comment by chrissm79** [comment]
Haven't dug into Laravel's route:list command, but you are correct, it was the `$relay->setupRequest($request->get('query'));` that was causing the issue. It's currently used for setting up middleware on queries. 

Just pushed a fix with `0.3.1`. Let me know if you still have issues. 

Thanks!
